Dockerfile

```diff
@@ -1,88 +1,110 @@
-FROM ruby:2.4.4-alpine3.6
+# Taken from Sir Boops' Dockerfile: https://git.sergal.org/Sir-Boops/docker-mastodon/src/branch/master/Dockerfile
+# For the purposes of using jemalloc.
 
-LABEL maintainer="https://github.com/tootsuite/mastodon" \
-      description="Your self-hosted, globally interconnected microblogging community"
+FROM ubuntu:18.04 as build-dep
 
-ARG UID=991
-ARG GID=991
+EXPOSE 3000 4000
```

Comment:
This is my only addition from Sir Boops' Dockerfile, linked to above. Not sure if it's necessary, I'm going to ping them and ask.